shared/main.lua

```diff
@@ -9,12 +9,12 @@ for i = 97, 122 do StringCharset[#StringCharset + 1] = string.char(i) end
 
 function QBShared.RandomStr(length)
     if length <= 0 then return '' end
-    return QBShared.RandomStr(length - 1) .. StringCharset[math.random(1, #StringCharset)]
+    return ('%s%s'):format(QBShared.RandomStr(length - 1), StringCharset[math.random(1, #StringCharset)])
 end
 
 function QBShared.RandomInt(length)
     if length <= 0 then return '' end
-    return QBShared.RandomInt(length - 1) .. NumberCharset[math.random(1, #NumberCharset)]
+    return ('%s%s'):format(QBShared.RandomInt(length - 1), NumberCharset[math.random(1, #NumberCharset)])
```

Comment:
```suggestion
    return QBShared.RandomInt(length - 1) .. NumberCharset[math.random(1, #NumberCharset)]
```
Maybe change this back, it's less efficient and also doesn't really improve readability.